trade-finance-manager-ui/server/controllers/helpers/amendments.helper.js

```diff
@@ -42,15 +42,16 @@ const userCanEditBankDecision = (amendment, user) => {
 const ukefDecisionRejected = (amendment) => {
   const { changeFacilityValue, changeCoverEndDate } = amendment;
   const { value, coverEndDate } = amendment.ukefDecision;
+  const { UNDERWRITER_MANAGER_DECISIONS } = DECISIONS;
```

Comment:
can we destructure the the UW_MANAGER_DECISION as well?
so something like this:
```js
const { DECLINED } = DECISIONS.UNDERWRITER_MANAGER_DECISIONS;
```

and then below in the return statement we can have

```js
return value === DECLINED && coverEndDate === DECLINED
```